### PR TITLE
Offer the possiblity to hide the recipient list in usercard (#2209)

### PR DESCRIPTION
**Pull request by olivierPigeon-RTE**
Signed-off-by: olivierPigeon-RTE <olivier.pigeon@rte-france.com>

In release notes, features section :
#2209 : Offer the possiblity to hide the recipient list in usercard

### CODE REVIEW

File: src/test/resources/bundles/userCardExamples3/config.json [hunk 1]
```diff
@@ -14,7 +14,8 @@
       "color": "#0070C0",
       "userCard" : {
         "template" : "usercard_task",
-        "lttdVisible" : false
+        "lttdVisible" : false,
+        "recipientVisible" : false
```
Comment (AlexGuironnetRTE) [inline src/test/resources/bundles/userCardExamples3/config.json:18]: I understand why we need a test with an empty recipientList, so OK for modifying the corresponding template under API, but not on this one because otherwise it makes the task usercard a bit useless to play with as it's only received by the sender themselves.

File: ui/main/src/app/modules/usercard/usercard.component.ts [hunk 2]
```diff
@@ -450,6 +451,7 @@ export class UserCardComponent implements OnDestroy, OnInit {
             this.startDateVisible = (userCard.startDateVisible === undefined) ? true : userCard.startDateVisible;
             this.endDateVisible = (userCard.endDateVisible === undefined) ? true : userCard.endDateVisible;
             this.lttdVisible = (userCard.lttdVisible === undefined) ? true : userCard.lttdVisible;
+            this.recipientVisible = (userCard.recipientVisible === undefined) ? true : userCard.recipientVisible;
```
Comment (AlexGuironnetRTE) [inline ui/main/src/app/modules/usercard/usercard.component.ts:454]: It's not specific to `recipientVisible`, but it looks like we're defining the default to be `true` 3 times:
- Line 111 when declaring the field
- Here, it the case where it's a usercard
- Line 463 if it's not a usercard

File: services/businessconfig/src/main/modeling/swagger.yaml [hunk 3]
```diff
@@ -537,6 +537,8 @@ definitions:
         type: boolean
       lttdVisible: 
         type: boolean
+      recipientVisible: 
+        type: boolean
```
Comment (AlexGuironnetRTE) [inline services/businessconfig/src/main/modeling/swagger.yaml:541]: It could be interesting to have a short description here like for other properties, so people wanting to send cards see at once what this property impacts without having to go look in the adoc documentation.
From the name alone it could mean displaying the recipients in the card rather than displaying the recipients dropdown.

**Review by AlexGuironnetRTE — CHANGES_REQUESTED** [review]
Tests all OK, but a few requested changes in code and doc. Let me know if anything's unclear!

File: ui/main/src/app/modules/usercard/usercard.component.ts (hunk 2 above)
Comment (olivierPigeon-RTE) [reply ui/main/src/app/modules/usercard/usercard.component.ts:454]: Ok, I'll let the default value of all fields at this place then (lines 449-456), and remove it from other places

**Review by AlexGuironnetRTE — APPROVED** [review]
